src/Providers/ScoutApmServiceProvider.php

```diff
@@ -38,8 +43,19 @@ final class ScoutApmServiceProvider extends ServiceProvider
     /** @throws BindingResolutionException */
     public function register() : void
     {
-        $this->app->singleton(self::CONFIG_SERVICE_KEY, static function () {
-            return Config::fromArray([]);
+        $this->app->singleton(self::CONFIG_SERVICE_KEY, function () {
+            $configRepo = $this->app->make(ConfigRepository::class);
+
+            return Config::fromArray(array_filter(array_combine(
+                ConfigKey::allConfigurationKeys(),
+                array_map(
+                    /** @return mixed */
+                    static function (string $configurationKey) use ($configRepo) {
+                        return $configRepo->get('scout.' . $configurationKey);
```

Comment:
@asgrim  this conflicts with Laravel scout config file :)

Reply:
I don't understand the comment, this is supposed to be reading the Scout config file, what do you mean it conflicts??

Reply:
scout apm using the name scout.php for configs conflicts with the config file of the first party package **Laravel/scout** https://github.com/laravel/scout

you guys should take such issues into consideration since [Laravel scout](https://github.com/laravel/scout) is a first-party package that has a huge usage.

Reply:
Thanks - created issue #37